test/e2e/e2e_test.go

```diff
@@ -97,7 +97,7 @@ func testWithOneSriovNetworkNodePolicyCR(t *testing.T, ctx *framework.TestCtx) {
 		Spec: sriovnetworkv1.SriovNetworkNodePolicySpec{
 			ResourceName: "resource-1",
 			NodeSelector: map[string]string{
-				"feature.node.kubernetes.io/sriov-capable": "true",
+				"feature.node.kubernetes.io/network-sriov.capable": "true",
```

Comment:
I see we use `sriov-capable` in `Readme` and `manifest/4.2`, is there particular reason to change the label name?
shall we change them all ?

Reply:
This label name is from NFD https://github.com/kubernetes-sigs/node-feature-discovery. NFD changed the label name, so we changed accordingly.

Regarding to README file, I will make some change in anther PR.